scrapers/va/bills.py

```diff
@@ -257,9 +277,11 @@ def add_votes(self, bill: Bill, legislation_id: str):
                 # in order to avoid duplicate dedupe keys
                 if not row["BatchNumber"]:
                     continue
+
                 v.dedupe_key = (
                     f"{row['BatchNumber'].strip()}-{bill.identifier.strip()}-"
                     f"{row['LegislationActionDescription'].strip()}"
+                    f"{row['LegislationActionDescription'].strip()}`"
```

Comment:
Why is `{row['LegislationActionDescription'].strip()}` being reported here (plus a backtick)?

Reply:
@jessemortenson , I removed this part in script and pushed the script again.
Please check it and let me know if you have any issues